src/serialize/window.js

```diff
@@ -213,7 +213,7 @@ export class ProxyWindow {
 
         const reopenPromise = ZalgoPromise.hash({ isPopup: isPopupPromise, name: getNamePromise }).then(({ isPopup, name }) => {
             if (isPopup && name) {
-                window.open('', name);
+                window.open('', name, 'noopener');
```

Comment:
Does this limit the behavior of the popup in any way? I noticed this message about Chromium on caniuse.com:
<img width="1430" alt="Screen Shot 2022-01-10 at 12 25 34 PM" src="https://user-images.githubusercontent.com/534034/148819039-94e61a1f-ff0c-49a2-8183-fd300128d688.png">

Reply:
`window.open` is being used here to bring focus back on the popup window.
So we might never need setting position or size features here. We should be able to safely ignore this chromium issue.